zynqmp/arm-trusted-firmware/arm-trusted-firmware.inc

```diff
@@ -0,0 +1,73 @@
+DESCRIPTION = "ARM Trusted Firmware"
```

Comment:
The project have changed name, It's "Trusted Firmware A" nowadays. So where possible I would write that instead and likewise for abbreviations, I would also change those where possible, i.e., from `ATF` to `TF-A` (or `TFA`).

Reply:
Don't want to change variable names as Petalinux defines some of them. project name updated.